### ISSUE TITLE
How to use the trained model in other environment of DeepMind Lab?

### ISSUE THREAD

**Issue by fanyuzeng** [zeus7777777/nav_a3c#2]
I trained the project,and get the trained model  .ckpt, .meta, . data. 
How to use those trained model in other environments of Deepmind lab?
I want to test the transfer ability of nav_a3c and acquire the reward in different environments.
Many thanks.

**Comment by zeus7777777** [comment]
Well, transferability is not mentioned in original paper.
If you really want to try that, you can modify line 12 in `environment.py` and uncomment line 34, 35 in `train.py`

**Comment by fanyuzeng** [comment]
Thank you, I try it now.

**Comment by fanyuzeng** [comment]
I ran the project with nav_maze_static_01 for 10104 episodes, and got trained model. 10104.cptk.data-00000-of-00001, model10104.cptk.index, model10104.cptk.meta .
Then I modified line 12 in environment.py, in which I change environment to nav_maze_static_02. And uncomment line 34, 35 in train.py. 
After above modification, I run the project, but it continued generate model whose id directly follows 10104, such as model10112.cptk.data-00000-of-00001,model10112.cptk.index, model10112.cptk.meta. 

I wonder why the model is generated when I used the trained model?
